marathon_lb.py

```diff
@@ -633,7 +641,9 @@ def config(apps, groups, bind_http_https, ssl_certs, templater):
             backend=backend,
             servicePort=app.servicePort,
             mode=app.mode,
-            sslCertOptions=' ssl crt ' + app.sslCert if app.sslCert else ''
+            sslCert=' ssl crt ' + app.sslCert if app.sslCert else '',
+            sslCertOptions=' ' + app.sslCertOptions if app.sslCertOptions else
+            ''
```

Comment:
pep8 check complained when it was all on one line. Don't know where to best break it though, this looks ugly to me, else on the next line didn't look better.